agenda/network/src/main/java/com/pronoidsoftware/agenda/network/KtorRemoteReminderDataSource.kt

```diff
@@ -27,6 +29,15 @@ class KtorRemoteReminderDataSource @Inject constructor(
         ).map { it.toReminder() }
     }
 
+    override suspend fun getAllReminders(): Result<List<Reminder>, DataError.Network> {
+        return httpClient.get<AgendaDto>(
+            route = AgendaRoutes.FULL_AGENDA,
+        )
+            .map { agendaDto ->
+                agendaDto.reminders.map { it.toReminder() }
+            }
+    }
+
```

Comment:
I'll replace this with mapping to all agenda item types, just wanted something for testing first